libkbfs/tlf_journal.go

```diff
@@ -366,10 +366,10 @@ func makeTLFJournal(
 	// Do this only once we're sure we won't error.
 	storedBytes := j.blockJournal.getStoredBytes()
 	if storedBytes > 0 {
-		availableBytes := j.diskLimiter.ForceAcquire(storedBytes)
+		j.diskLimiter.onJournalEnable(storedBytes)
 		j.log.CDebugf(ctx,
-			"Force-acquired %d bytes for %s: available=%d",
-			storedBytes, tlfID, availableBytes)
+			"Force-acquired %d bytes for %s: %v",
+			storedBytes, tlfID, j.diskLimiter)
```

Comment:
Why not continue to print out `availableBytes`?  Was this for debugging? Doesn't seem too useful to print out the interface here, since there's no `String()` on the `diskLimiter` interface.